main.go

```diff
@@ -18,14 +19,13 @@ type Main struct {
 
 func main() {
 	var mainObj Main = NewMain()
+	mainObj.logger.Println("Initialized main.")
 
-	var workflow workflows.InterfaceWorkflow = mainObj.workflowFactory.constructWorkflow("dummy_workflow")
-	var workflowContext *nodes.WorkflowContext = nodes.NewWorkflowContext()
-	workflow.Run(workflow, workflowContext)
-
-	for index, node := range workflow.GetNodes() {
-		mainObj.logger.Println(fmt.Sprintf("Result for node index %d=%s", index, node.GetResult(node, workflowContext).(string)))
-	}
+	// Setup main such that main does not exit unless there is a keyboard interrupt.
+	go forever()
```

Comment:
Even without this, main will not exit because of the channel we created unless it is interrupted.  so we can remove this forever method.

Reply:
I see, I have removed the forever method.